selenium-config.template.toml

```
@@ -0,0 +1,11 @@
+[distributor]
+reject-unsupported-caps = true # Reject a request immediately if the Grid does not support the requested capability.
+
+[node]
+detect-drivers = false # Ignore local drivers and use the ones specified in the driver configuration section.
+
+[[node.driver-configuration]]
+display-name = "chrome"
+max-sessions = 2
+webdriver-path = "./chromedriver"
+stereotype = '{"browserName": "chrome", "browserVersion": "v120.0"}' # Browser version is not used by the Grid, but it is used by the tests to determine which browser to use.
```

Hmm, if the user's local chrome version is different than this, will things break?

---

Oh, I think this should be
```suggestion
stereotype = '{"browserName": "chrome", "browserVersion": "v$full_chrome_version"}' # Browser version is not used by the Grid, but it is used by the tests to determine which browser to use.
```

or something like that

---

Addressed in [bc4c42f](https://github.com/iFixit/selenium_setup/pull/17/commits/bc4c42f5a25ad7b86315f20b71ff634a8fd19896)